components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -296,37 +290,45 @@ end
 function CustomLeague:_makeVersionLink(args)
 	if String.isEmpty(args.version) then return nil end
 	local content = GameLookup.getName({args.game}) .. '/' .. args.version
-	content = content .. '|' .. args.version
 
-	return Page.makeInternalLink(content)
+	return Page.makeInternalLink(args.version, content)
 end
 
 function CustomLeague:_makePatchLink(args)
 	if String.isEmpty(args.patch) then return nil end
 
 	local content
 	local patch =  GameLookup.getName({args.game}) .. '/' .. args.version .. '/' .. args.patch
-	content = Page.makeInternalLink(patch, args.patch)
+	content = Page.makeInternalLink(args.patch, patch)
 
 	if not String.isEmpty(args.epatch) then
 		content = content .. '&nbsp;&ndash;&nbsp;'
 		local epatch = GameLookup.getName({args.game}) .. '/' .. args.version .. '/' .. args.epatch
-		epatch = Page.makeInternalLink(epatch, args.epatch)
+		epatch = Page.makeInternalLink(args.epatch, epatch)
 		content = content .. epatch
 	end
 
 	return content
 end
 
-function CustomLeague:_getGameModes(args)
+function CustomLeague:_getGameModes(args, makeLink)
+	makeLink = makeLink or false
```

Comment:
this line is not needed
for ifs `nil` has the same effect as `false`

Reply:
removed